README.md

```diff
@@ -1 +1,13 @@
+# Chef Jump-In Site
+
+A simple website to list Github issues labelled with `Type: Jump In`.
+
+## Building Locally
+
+- Install node and yarn. On a mac this can be done via brew with `brew install node yarn`
+- Install the node packages via yarn with `yarn install` from the repo directory
```

Comment:
This is missing some steps:

 git clone git@github.com:chef/jump-in.git
 cd jump-in

and then yarn install (or npm install)